### ISSUE TITLE
The generated QR is not correct

### ISSUE THREAD

**Issue by ruben-monmariola** [bashkirtsevich/Pascal-QRCode#1]
The generated QR codes cannot be read with a QR code reader.
I have been comparing the generated images with standard QR codes and the text is encoded correctly, but the information of the EC Level, mask pattern and error correction level indicated in the QR code are incorrect.
I honestly don't know how to solve this problem.

**Comment by AlexTuduran** [comment]
Yes, found it out the hard way. Should have read the issues first. What a pitty.
